EDMarketConnector.py

```diff
@@ -330,6 +330,30 @@ def __init__(self, master: tk.Tk):  # noqa: C901, CCR001 # TODO - can possibly f
 
         self.prefsdialog = None
 
+        if platform == 'win32':
+            import threading
+            from infi.systray import SysTrayIcon
+
+            self.only_tray_close = 0  # This is kind of a hack.
```

Comment:
I'm still not seeing why this is an `int` instead of a `bool`.  The config.py code will be doing the conversion to/from `int` for Windows Registry purposes.

Reply:
And now I read the comment about the hack... which I'm going to have to have a think about.

Reply:
Did you mark the wrong piece of code? Cause the marked code has nothing to do with config.

Reply:
Ah yes, but my point about using an Enum still stands.